### ISSUE TITLE
Add done to test-app methods.

### ISSUE THREAD

**Issue by cammoore** [radgrad/radgrad2#59]
Please do work for this task in a branch called issue-59.

On Ubuntu we get a time out for the test-app. Maybe adding done to all the it methods will solve this problem.

**Comment by gcalica** [comment]
Mine also times out for either test or test-app.
Sometimes it straight up freezes on some text that says "X passing (Ys)". If that does happen, it's always preceded by this error message: [Img](https://imgur.com/a/Put8ziT)

Or the tests run all the way through but it says I have a few server failures. And the failures are always some LevelProcessor timing out after exceeding 5000ms.

**Comment by gcalica** [comment]
When executing `meteor npm run test`,
https://imgur.com/a/8uGYAiB
Fixed in https://github.com/radgrad/radgrad2/commit/fd0c3ec9f3c46c08412fba2ec51efdeb55487308 by raising timeout to 15000ms. (Couldn't see the execution time of when it failed so I just used 15000ms).

**Comment by gcalica** [comment]
When executing `meteor npm run test-app`,
https://imgur.com/a/5BpkItS

**Comment by qauchida** [comment]
When executing `meteor npm run test-app`, https://drive.google.com/file/d/1su7M1CRTxHHIzDW7vJTEkYRXFwl61hyz/view?usp=sharing

**Comment by cammoore** [comment]
After reading a bunch of blog posts about getting Nightmare to run headless on linux. I found out that Nighmare needs xvfb to run. So I installed it and all its dependencies. 

Run

sudo apt-get install -y xvfb x11-xkb-utils xfonts-100dpi xfonts-75dpi xfonts-scalable xfonts-cyrillic x11-apps clang libdbus-1-dev libgtk2.0-dev libnotify-dev libgnome-keyring-dev libgconf2-dev libasound2-dev libcap-dev libcups2-dev libxtst-dev libxss1 libnss3-dev gcc-multilib g++-multilib

This fixed the issue on my Ubuntu laptop.
